uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java

```diff
@@ -64,8 +64,12 @@ public class BrokerCompanyPortal extends AbstractTestBroker implements ITestBrok
     public final static String COMPANY_PORTAL_APP_NAME = "Intune Company Portal";
     public final static String COMPANY_PORTAL_APK = "CompanyPortal.apk";
     public final static String OLD_COMPANY_PORTAL_APK = "OldCompanyPortal.apk";
+    private final static int PASSWORD_UI_ATTEMPT_COUNT = 3;
+
+    final static long ENROLLMENT_PAGE_TIMEOUT = TimeUnit.SECONDS.toMillis(45);
 
     private boolean enrollmentPerformedSuccessfully;
+    private boolean batteryOptimizationTurnedOff;
```

Comment:
I think you used the word `disabled` some places and `turnedOff` some places. Let's make these consistent. I think `disabled` might be better as less characters but I'd leave it up to you.

Reply:
will add consistency here. I used `turnedoff` because that's the terminology used in CP UI.